collect_app/src/main/java/org/odk/collect/android/adapters/FileArrayAdapter.java

```diff
@@ -76,10 +82,16 @@ void onBind(DriveListItem item) {
                 Drawable d = ContextCompat.getDrawable(getContext(), R.drawable.form_state_blank);
                 imageView.setImageDrawable(d);
                 checkBox.setVisibility(View.VISIBLE);
+                if (isNewVersion(item)) {
+                    formUpdateAlert.setVisibility(View.VISIBLE);
+                } else {
+                    formUpdateAlert.setVisibility(View.GONE);
```

Comment:
If gone is needed at all here (depends on my question below since it;s the same case) I would prefer using ternary operator: `formUpdateAlert.setVisibility(isNewVersion(item) ? View.VISIBLE : View.GONE);`